recipe/meta.yaml

```diff
@@ -1,10 +1,10 @@
 {% if version is not defined %}
-{% set version = "7.3.13" %}
+{% set version = "7.3.15" %}
 {% endif %}
 {% if python_version is not defined %}
 {% set python_version = "3.9.18" %}
 {% endif %}
-{% set cffi_version = "1.15.1" %}
+{% set cffi_version = "1.16.0" %}
 {% set python_maj_min = ".".join(python_version.split(".")[:2]) %}
 # Keep increasing this until we get a new version, python_version and cffi_version. Do not reset
 {% set build_num = "0" %}
```

Comment:
Please follow the comment here and increase the build number

Reply:
Doesn't this PR count as "get a new version" and "get a new cffi_version"?

Reply:
The package names on https://anaconda.org/conda-forge/pypy3.9/files include 7.3.15 instead of 7.3.13, and the hashes are different. Isn't that enough to reset the `build_num`?

Reply:
The comment explicitly mentions `python_version`, which isn't changed here. That means there will be a new build of python 3.9.18, which should get a new vuild number (it'll get a new hash due to now host-depending on a newer pypy, but we should still increase the build number).

Unless we split up the build numbers to increment them separately (more work and easy to get wrong), the build number should _only_ be reset if all three mentioned versions increase simultaneously.

Reply:
Ahh, "and" not "or". Thanks.